### PR TITLE
RefMan-ext: fix some typos

### PR DESCRIPTION
**Pull request by william-lawvere**
While reading the documentation on Function, I noticed some typos which I fixed.

### CODE REVIEW

**Comment by herbelin** [comment]
Maybe @forestjulien or @Matafou want to confirm, but as far as I'm concerned, thanks a lot for these typos, and I see no reason to delay the integration.

**Comment by Matafou** [comment]
No objection. Thanks a lot!
Pierre

Le jeu. 13 juil. 2017 à 12:54, Hugo Herbelin <notifications@github.com> a
écrit :

File: doc/refman/RefMan-ext.tex [hunk 1]
```diff
@@ -705,18 +705,18 @@ \section{Advanced recursive functions}
...
+given (unless the function is not recursive), but it might not
+necessarily be \emph{structurally} decreasing. The point of the {\tt
   \{\}} annotation is to name the decreasing argument \emph{and} to
 describe which kind of decreasing criteria must be used to ensure
 termination of recursive calls.
 
-The {\tt Function} construction enjoys also the {\tt with} extension
+The {\tt Function} construction also enjoys the {\tt with} extension
 to define mutually recursive definitions. However, this feature does
 not work for non structural recursive functions. % VRAI??
```
Comment (psteckler) [inline doc/refman/RefMan-ext.tex]: "non-structurally recursive" maybe?

**Comment by psteckler** [comment]
I added one small comment.

File: doc/refman/RefMan-ext.tex (hunk 1 above)
Comment (william-lawvere) [inline doc/refman/RefMan-ext.tex]: Changed to non structurally recursive (I didn't use a hyphen, to match "For now dependent cases are not treated for non structurally terminating functions." later on in the file)

**Comment by ecvideo** [comment]
Envoyé de mon smartphone BlackBerry 10.                                                                                                                                                                                                                De: william-lawvereEnvoyé: samedi 15 juillet 2017 02:15À: coq/coqRépondre à: coq/coqCc: SubscribedObjet: Re: [coq/coq] RefMan-ext: fix some typos (#865)@william-lawvere commented on this pull request.



In doc/refman/RefMan-ext.tex:
>    \{\}} annotation is to name the decreasing argument \emph{and} to
 describe which kind of decreasing criteria must be used to ensure
 termination of recursive calls.
 
-The {\tt Function} construction enjoys also the {\tt with} extension
+The {\tt Function} construction also enjoys the {\tt with} extension
 to define mutually recursive definitions. However, this feature does
 not work for non structural recursive functions. % VRAI??

Changed to non structurally recursive (I didn't use a hyphen, to match "For now dependent cases are not treated for non structurally terminating functions." later on in the file)

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/coq/coq","title":"coq/coq","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/coq/coq"}},"updates":{"snippets":[{"icon":"PERSON","message":"@william-lawvere commented on #865"}],"action":{"name":"View Pull Request","url":"https://github.com/coq/coq/pull/865#discussion_r127570249"}}}

**Comment by maximedenes** [comment]
Merged in master, thanks!

**Pull request merged**